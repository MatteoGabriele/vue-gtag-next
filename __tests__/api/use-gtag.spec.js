import useGtag from "@/api/use-gtag";

jest.mock("@/api", () => ({
  foo: 1,
  bar: 2,
}));

describe("useGtag", () => {
  it("should return the api object", () => {
    expect(useGtag()).toMatchSnapshot();
  });
});
