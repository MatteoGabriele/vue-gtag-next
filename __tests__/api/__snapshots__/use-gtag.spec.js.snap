// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useGtag should return the api object 1`] = `
Object {
  "bar": 2,
  "default": Object {
    "bar": 2,
    "foo": 1,
  },
  "foo": 1,
}
`;
